### ISSUE TITLE
Need help in modifying Ping Sitemap URL

### ISSUE THREAD

**Issue by thegulshankumar** [rankmath/seo-by-rank-math#7]
TLDR: When the deployment is triggered by static HTML output plugin, ping URL to search engine

My WordPress exists at my computer `http://localhost` (uses Rankmath, [static html output](https://github.com/leonstafford/static-html-output)) for **daily** updating blog at `https://www.example.com` 

I am [aware](https://support.rankmath.com/ticket/how-to-refresh-sitemap/#post-795) that when we publish post or update, Rankmath ping to Google.

Is there any function that can help in Pinging production URL `https://www.example.com` from the localhost server after successful deployment from 'static HTML output plugin' plugin.

Related https://github.com/rankmath/seo-by-rank-math/blob/653b0e055c5cecf02dfc683f7964f1e1db121ad0/includes/modules/sitemap/class-sitemap.php#L147

Thanks

**Comment by surajv** [comment]
Hi Gulshan,

You can try hooking this function in your deployment event:

`RankMath\Sitemap::ping_search_engines( 'live_site_post_url' );`

Please note, `live_site_post_url` needs to be replaced with the real URL.

**Comment by leonstafford** [comment]
@thegulshankumar the hook you'd want to use if trying this has an [example here](https://github.com/leonstafford/static-html-output#post-deployment-hook)
